npm/index.js

```diff
@@ -15,4 +15,20 @@ function getElectronPath () {
   }
 }
 
-module.exports = getElectronPath()
+// A list of the main modules the people will attempt to use from the Electron API, this is not a complete list but should cover most
+// use cases.
+var electronModuleNames = ['app', 'autoUpdater', 'BrowserWindow', 'ipcMain', 'Menu', 'net', 'Notification', 'systemPreferences', 'Tray']
+var electronPath = new String(getElectronPath())
+
+electronModuleNames.forEach(function warnOnElectronAPIAccess (apiKey) {
+  Object.defineProperty(electronPath, apiKey, {
+    enumerable: false,
+    configurable: false,
+    get: function getElectronAPI () {
+      console.warn('WARNING: You are attempting to access an Electron API from a node environment.\n\n' +
+      'You need to use the "electron" command to run your app. E.g. "electron ./myapp.js"')
```

Comment:
Perhaps suggest `npx electron`?

Reply:
`npx electron` will end up using a global Electron right? We want to avoid people having Electron installed globally as that causes other issues where people end up using the wrong version of Electron 🤔

Reply:
`npx` uses `node_modules/.bin` first: https://www.npmjs.com/package/npx#description

Reply:
that's not much shorter than `node_modules/.bin/electron` and the latter is much simpler to understand